bundles/InstallBundle/Resources/install.sql

```diff
@@ -879,6 +879,7 @@ CREATE TABLE `gridconfig_favourites` (
     PRIMARY KEY (`ownerId`, `classId`, `searchType`, `objectId`),
 	INDEX `classId` (`classId`),
 	INDEX `searchType` (`searchType`),
+    INDEX `grid_config_id` (`gridConfigId`),
```

Comment:
The other indexes are in camel case, why not this also?

Reply:
So that constraint references column instead of index. Otherwise you won't be able to delete index in down migration.

Reply:
Okay, thanks for clarifying